widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte

```diff
@@ -12,7 +12,7 @@
 	<svelte:fragment slot="before">
 		<input
 			bind:group
-			class="bg-gray-200 border-transparent focus:border-transparent text-blue-500 focus:ring-1 focus:ring-offset-2 focus:ring-gray-200 mr-2"
+			class="bg-gray-200 dark:bg-gray-700 shadow-inner border-transparent focus:border-transparent text-blue-500 ring-1 ring-offset-2 ring-gray-200 dark:ring-gray-700 mr-2"
```

Comment:
What's the reason behind this change ?

Reply:
Two reasons:
1. The radio was looking very flat and unrecognizable. By enabling the ring and adding inner-shadow it definitely looks like a radio
2. Dark mode support

Reply:
Alright ! I now see it in the screenshots. Tagging @gary149 so he can have a look if he wants to